components/OtherResourcesForm/OtherResourcesForm.tsx

```diff
@@ -1,27 +1,166 @@
-import clsx from "clsx";
-import { useTranslation } from "react-i18next";
-import Textarea from "../Form/Textarea";
+import clsx from 'clsx'
+import { useTranslation } from 'react-i18next'
+import Textarea from '../Form/Textarea'
+import { useOthersForm } from '@/hooks/useData'
+import { useForm } from 'react-hook-form'
+import DateInput from '@/components/Form/Date'
+import Dropdown from '@/components/Form/Dropdown'
+import Input from '@/components/Form/Input'
+import endpoints from '../../endpoints.json'
+import { useState } from 'react'
 
 const OtherResourcesForm = ({}) => {
-  const { t } = useTranslation();
+  const { t } = useTranslation()
+  const { data: formData } = useOthersForm()
+  const [serverErrors, setServerErrors] = useState<{ [key: string]: string[] }>(
+    {}
+  )
+  const countyCovarage = formData ? formData['county_coverage'].choices : []
+  const today = new Date().toISOString().substr(0, 10)
+  const {
+    handleSubmit,
+    register,
+    formState: { errors },
+  } = useForm()
 
-  return( 
-  <section
-  className={clsx(
-    'container grid place-items-start',
-    'bg-blue-50 rounded',
-    'mx-auto px-8 py-7',
-  )}
-  >
-    <h3 className="text-sm">{`${t('describeTheResource')}:`}</h3>
-    <Textarea 
-    name="Other resources"
-    className={clsx(
-      'w-full'
-    )}
-    />
-  </section>
-  );
-};
+  const onSubmit = async (values: any) => {
+    const res = await fetch(
```

Comment:
we need to extract this so we also pass the token. Should we add axios and make use of interceptors?
Just sparking a discussion

Reply:
yep, my plan was to extract them when we have everyting working.. i don't mind to keep them raw, and when we have a bigger picture of the requests and responses look like we can add adaptors and all nice abstractions :D